api/v1/db_types.go

```diff
@@ -159,3 +159,47 @@ func (s Summary) GormValue(ctx context.Context, db *gorm.DB) clause.Expr {
 	data, _ := json.Marshal(s)
 	return gorm.Expr("?", data)
 }
+
+type LogsSelectors []LogsSelector
+
+func (rs LogsSelectors) Value() (driver.Value, error) {
+	if len(rs) == 0 {
+		return []byte("[]"), nil
+	}
+
+	return json.Marshal(rs)
+}
+
+func (rs *LogsSelectors) Scan(val interface{}) error {
+	if val == nil {
+		*rs = LogsSelectors{}
+		return nil
+	}
+
+	var ba []byte
+	switch v := val.(type) {
+	case []byte:
+		ba = v
+	default:
+		return fmt.Errorf("value is not []byte: It's %T", val)
+	}
+
+	return json.Unmarshal(ba, rs)
+}
```

Comment:
You can use duty's generic types like we do in incident commander
```
import github.com/flanksource/duty/types


func (p PersonProperties) Value() (driver.Value, error) {
	return types.GenericStructValue(p, true)
}

func (p *PersonProperties) Scan(val any) error {
	return types.GenericStructScan(&p, val)
}
```

Ref: https://github.com/flanksource/incident-commander/blob/3260cdebf9c498017df13715ff95f4adf174be85/api/team.go#L50-L56